src/containers/board-container.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Light } from '../components/light.js';
+import { Board } from '../components/board.js';
+
+export class BoardContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            board: [],
+            numMoves: 0,
+            winComponent: <p />,
+        };
+        this.board = this.getNewBoard();
+    }
+
+    /**
+     * Decides which tiles start off in the On positon.
+     * If no lights get turned on, the method will call itself again.
+     */
+    getNewBoard = () => {
+        let board = [];
+        let isALightOn = false;
+        for (let row = 0; row < 5; row++) {
+            board.push([]);
+            for (let col = 0; col < 5; col++) {
+                let isOn = Math.random() < 0.1 ? true : false;
+                if (isOn) isALightOn = true;
+                board[row].push(isOn);
+            }
+        }
+        if (isALightOn) {
```

Comment:
There's a super-low chance of an all-dark board being produced, but it's good to check for it! Another way to check would be to look at the board itself instead of saving values during the loop, like so:

```js
const isALightOn = board.every(row => row.every(value => value))
```